### PR TITLE
Add successful stopped workspaces metric

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Add counter for successful stops of workspace

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13270

### CODE REVIEW

File: wsmaster/che-core-api-metrics/src/main/java/org/eclipse/che/api/metrics/WorkspaceSuccessfulStopAttemptsMeterBinder.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  public void bindTo(MeterRegistry registry) {
+    stoppedCounter =
+        Counter.builder(workspaceMetric("stopped.total"))
+            .description("The count of stopped workspaces")
+            .register(registry);
+
+    // only subscribe to the event once we have the counters ready
+    eventService.subscribe(
+        event -> {
+          if (event.getPrevStatus() == WorkspaceStatus.STOPPING
```
Comment (sleshchenko) [inline wsmaster/che-core-api-metrics/src/main/java/org/eclipse/che/api/metrics/WorkspaceSuccessfulStopAttemptsMeterBinder.java]: I guess you should also check if an error is empty

**Comment by skabashnyuk** [comment]
Docs? Grafana panel?

File: wsmaster/che-core-api-metrics/src/main/java/org/eclipse/che/api/metrics/WorkspaceSuccessfulStopAttemptsMeterBinder.java (hunk 1 above)
Comment (mkuznyetsov) [reply wsmaster/che-core-api-metrics/src/main/java/org/eclipse/che/api/metrics/WorkspaceSuccessfulStopAttemptsMeterBinder.java]: yes, thx

**Comment by mkuznyetsov** [comment]
Screenshot from 2019-05-27 11-12-16
grafana panel preview, and PR that adds it to docs:

https://github.com/eclipse/che-docs/pull/721

**Review by skabashnyuk — APPROVED** [review]

**Comment by mkuznyetsov** [comment]
ci-test

**Comment by SkorikSergey** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1816//Selenium_20tests_20report/) doesn't show any regression against this Pull Request.

**Pull request merged**